geometry_msgs/Point Xs
float64[9] covariance